### ISSUE TITLE
[HW3] /articles 페이지에서 article id, author name, article title을 보여주는 방식에 대해 질문입니다.

### ISSUE THREAD

**Issue by ChanwooLucioLee** [swsnu/swppfall2020#101]
조교님 안녕하세요, /articles 페이지의 설명에서  article title이 해당 article의 detail page로 넘어가도록 button tag으로 감싸져야 한다는 것 외에 각 article의 article id, author name, article title이 어떤 방식으로 render되어야 하는 지 궁금해서 질문드립니다.

예를 들어서 그냥 매우 단순하게 {article id} {author name} <button>{article title}</button> 이와 같은 식으로 아무튼 포함만 되도록 render하면 충분한 것인가요? 다른 페이지의 html elements는 다 특정한 id를 가진 tag 사이로 감싸지도록 설명이 나와 있는데 여기에서는 그런 설명이 안 나와 있어서 자유롭게 해도 괜찮은 것인가 궁금했습니다.

**Comment by hy00nc** [comment]
네, /articles의 경우에는 article에 대한 정보들이 모두 들어가있게 render되기만 하면 됩니다. 버튼의 경우 title 내용이 잘 들어가있으면 됩니다.

**Comment by ChanwooLucioLee** [comment]
네 알겠습니다. 답변 감사드립니다.
